eng/.docsettings.yml

```diff
@@ -8,9 +8,13 @@ omitted_paths:
   - sdk/*/*.Management.*/*
   - samples/*
   - common/SmokeTests/*
-  - sdk/*/samples/*
+  - sdk/*/*/samples/*
   - sdk/*/swagger/*
-  - doc/ApiDocGeneration/*
+  - doc/*
+  - sdk/search/
+  - sdk/keyvault/
```

Comment:
Shouldn't these paths still be associated with a bug as they should each have a README in their root folder.

Reply:
I actually deliberately removed them from the buglist when I added them to the omission list. If they should be associated with the bug, I'll add the items back. 

I am adding them to the omission list deliberately based on the discussion that caused this issue to be opened in the first place :P